examples/platforms/cc2538/README.md

```diff
@@ -22,6 +22,7 @@ In a Bash terminal, follow these instructions to build the cc2538 examples.
 
 ```bash
 $ cd <path-to-openthread>
+$ ./script/bootstrap
```

Comment:
Yeah, that makes sense. I noticed that there is some inconsistencies in the format and content of these example READMEs (i.e. the cc256x vs nrf52840), even though they depend on the same toolchain. Should we work to make them cohesive and identical?